packages/dataviews/src/dropdown-menu-helper.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * WordPress dependencies
+ */
+import {
+	Icon,
+	privateApis as componentsPrivateApis,
+} from '@wordpress/components';
+import { forwardRef } from '@wordpress/element';
+import { SVG, Circle } from '@wordpress/primitives';
+
+/**
+ * Internal dependencies
+ */
+import { unlock } from './lock-unlock';
+
+const { DropdownMenuItemV2Ariakit: DropdownMenuItem } = unlock(
+	componentsPrivateApis
+);
+
+const radioCheck = (
+	<SVG xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+		<Circle cx={ 12 } cy={ 12 } r={ 3 }></Circle>
+	</SVG>
+);
+
+/**
+ * A custom implementation of a radio menu item using the standard menu item
+ * component, which allows deselecting selected values.
+ */
+export const DropdownMenuRadioItemCustom = forwardRef(
```

Comment:
If we pass a named function to the `forwardRef`, React devtools display it accordingly, so it's easier to search for. Arguably a nitpick, though I find it easier to have proper names for debugging. While I was looking at this component for other reasons, I went ahead and updated it at https://github.com/WordPress/gutenberg/pull/57416